src/utils/transform-filters.ts

```diff
@@ -0,0 +1,71 @@
+import { CombinedFilter, Filter, FilterCombinator } from '@yext/answers-core';
+import { SelectableFilter } from '../models/utils/selectablefilter';
+
+function formatOrFilters(filters: Filter[]): Filter | CombinedFilter {
```

Comment:
`formatOrFilters` makes me think that we are passing in 'Or' filters to this function and formatting them. How about a name such as 'combineFiltersWithOR'?

Reply:
make sense. Update!